### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by hari-ajidesh**
Closes #14 by adding the link to CONTRIBUTING file to the README file.

### CODE REVIEW

**Comment by hari-ajidesh** [comment]
Please add the hacktoberfest-accepted label to this issue

**Comment by kailashchoudhary11** [comment]
> Please add the hacktoberfest-accepted label to this issue

There is no need to add hacktoberfest-accepted label as the hacktoberfest topic is already added to the repo!
Also Please use this format for linking the file ``[Description](Filename)``

**Comment by hari-ajidesh** [comment]
The requested update has been done.

File: README.md [hunk 1]
```diff
@@ -120,68 +120,10 @@ This project, ***imgman***, is a project initiative founded by <https://github.c
 
 > Project is : Development
 
-## Running Locally
+## [Contribution Rules and Regulations](https://github.com/kailashchoudhary11/imgman/blob/master/CONTRIBUTING.md)
```
Comment (kailashchoudhary11) [inline README.md]:
```suggestion
## Contributing To The Project
```

File: README.md [hunk 2]
```diff
@@ -120,68 +120,10 @@ This project, ***imgman***, is a project initiative founded by <https://github.c
...
 
-### Prerequisites
+The proceedure for installation and usage is also provided here. Do carefully read through the instructions.
```
Comment (kailashchoudhary11) [inline README.md]:
```suggestion
In order to contribute to the project and setup the project locally refer to [Contributing](CONTRIBUTING.md) File.
```

**Review by kailashchoudhary11 — CHANGES_REQUESTED** [review]
Kindly make these changes.

**Comment by hari-ajidesh** [comment]
Requested changes are committed.

**Comment by kailashchoudhary11** [comment]
Thanks. I am merging the PR. Don't Forget To Give a Start to the Repository!

**Pull request merged**